openrouteservice-api-tests/src/test/java/org/heigit/ors/v2/services/routing/ParamsTest.java

```diff
@@ -700,6 +701,53 @@ public void expectAvoidpolygonsTypeError() {
 				.statusCode(400);
 	}
 
+	@Test
+	public void expectAvoidpolygonsRejectTooLargePolygons() {
+		JSONObject body = new JSONObject();
+		body.put("coordinates", getParameter("coordinatesShort"));
+		body.put("preference", "shortest");
+
+		JSONObject avoidGeom = new JSONObject("{\"type\":\"Polygon\",\"coordinates\":[[[0,0],[0.05,0],[0.05,0.001],[0,0.001],[0,0]]]}");
+		JSONObject options = new JSONObject();
+		options.put("avoid_polygons", avoidGeom);
+		body.put("options", options);
+
+		given()
+				.header("Accept", "application/json")
+				.header("Content-Type", "application/json")
+				.pathParam("profile", getParameter("carProfile"))
+				.body(body.toString())
+				.when()
+				.post(getEndPointPath() + "/{profile}")
+				.then()
+				.assertThat()
+				.body("error.code", is(2003))
+				.body("error.message", is("The extent of a polygon to avoid must not exceed 1.5 kilometers."))
+				.statusCode(400);
+
+		body = new JSONObject();
+		body.put("coordinates", getParameter("coordinatesShort"));
+		body.put("preference", "shortest");
+
+		avoidGeom = new JSONObject("{\"type\":\"Polygon\",\"coordinates\":[[[0,0],[0.1,0],[0.1,0.01],[0,0.01],[0,0]]]}");
+		options = new JSONObject();
+		options.put("avoid_polygons", avoidGeom);
+		body.put("options", options);
+
+		given()
+				.header("Accept", "application/json")
+				.header("Content-Type", "application/json")
+				.pathParam("profile", getParameter("carProfile"))
+				.body(body.toString())
+				.when()
+				.post(getEndPointPath() + "/{profile}")
+				.then().log().all()
+				.assertThat()
+				.body("error.code", is(2003))
+				.body("error.message", is("The area of a polygon to avoid must not exceed 2.5 square kilometers."))
```

Comment:
Shouldn't the 2.5 sqkm be drawn from the config?

Reply:
ditto